oe_whitelabel.theme

```diff
@@ -155,6 +155,23 @@ function oe_whitelabel_preprocess_page(&$variables) {
   $variables['site_logo_href'] = $site_logo_href;
 }
 
+/**
+ * Implements hook_preprocess_html().
+ */
+function oe_whitelabel_preprocess_html(array &$variables) {
+  $cck_container = [
+    '#type' => 'html_tag',
+    '#tag' => 'div',
+    '#attributes' => [
+      'id' => 'cck_here',
+      'role' => 'alertdialog',
```

Comment:
lets stick to the ticket as discussed in the daily and use "alert" and remove the "aria-live"
"aria-live="assertive" should only be used for time-sensitive/critical notifications that absolutely require the user's immediate attention. " so dont think it is what we need